[2025-02-23T04:35:48.784+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-23T04:35:48.823+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v1.first_task scheduled__2022-02-07T00:00:00+00:00 [queued]>
[2025-02-23T04:35:48.844+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v1.first_task scheduled__2022-02-07T00:00:00+00:00 [queued]>
[2025-02-23T04:35:48.845+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 6
[2025-02-23T04:35:48.989+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): first_task> on 2022-02-07 00:00:00+00:00
[2025-02-23T04:35:49.003+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=6017) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-23T04:35:49.002+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v1', 'first_task', 'scheduled__2022-02-07T00:00:00+00:00', '--job-id', '1355', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpcsiqzuw7']
[2025-02-23T04:35:49.006+0000] {standard_task_runner.py:72} INFO - Started process 6029 to run task
[2025-02-23T04:35:49.006+0000] {standard_task_runner.py:105} INFO - Job 1355: Subtask first_task
[2025-02-23T04:35:49.108+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag_v1.first_task scheduled__2022-02-07T00:00:00+00:00 [running]> on host f66dd6409c88
[2025-02-23T04:35:49.271+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='vn5705j' AIRFLOW_CTX_DAG_ID='our_first_dag_v1' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2022-02-07T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-02-07T00:00:00+00:00'
[2025-02-23T04:35:49.273+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-23T04:35:49.307+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-02-23T04:35:49.309+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', "echo 'Hello World!' This is the first Task"]
[2025-02-23T04:35:49.328+0000] {subprocess.py:99} INFO - Output:
[2025-02-23T04:35:49.330+0000] {subprocess.py:106} INFO - Hello World! This is the first Task
[2025-02-23T04:35:49.330+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-02-23T04:35:49.395+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-23T04:35:49.396+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v1, task_id=first_task, run_id=scheduled__2022-02-07T00:00:00+00:00, execution_date=20220207T000000, start_date=20250223T043548, end_date=20250223T043549
[2025-02-23T04:35:49.671+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-23T04:35:49.720+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-23T04:35:49.727+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
